.platform/solr_config/solrcore.properties

```diff
@@ -1,16 +1,14 @@
 solr.replication.master=false
 solr.replication.slave=false
 solr.replication.pollInterval=00:00:60
-solr.replication.masterUrl=http://solr.internal:8080/solr
+solr.replication.masterUrl=http://localhost:8983/solr
```

Comment:
Is this right? Platform.sh would usually use port 8080 (hostname might vary with the service name change?) but this one looks like a Lando service which typically runs on port 8983

Reply:
This is identical to the comment I left on Martin's PR and this morning on standup he said that 8983 oddly enough works.

Reply:
Okay, great. If it works it works :)